epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyViewHolder.java

```diff
@@ -43,7 +46,7 @@ public void bind(@SuppressWarnings("rawtypes") EpoxyModel model,
     this.payloads = payloads;
 
     if (epoxyHolder == null && model instanceof EpoxyModelWithHolder) {
-      epoxyHolder = ((EpoxyModelWithHolder) model).createNewHolder();
+      epoxyHolder = ((EpoxyModelWithHolder) model).createNewHolder(parent);
```

Comment:
what do you think about nulling out `parent` after this `if` statement so we don't leak a reference? If the recycled view pool is used across multiple recyclerviews (like with the activity wide pool) then there could be multiple parents, and if one is destroyed i wouldn't want it to leak here.

It's a little funky, but I think it would be safe to null it out

Reply:
Ok, because it is little funky I will add a comment.